server.js

```diff
@@ -20,9 +22,16 @@ app.use(express.static(__dirname + '/public'));
 // Log every request to the console
 app.use(morgan('dev'));
 
+app.use(staticServer(path.resolve(__dirname, 'public')));
+
 // routes
 require('./app/routes')(app);
 
+app.use(function (err, req, res, next) {
```

Comment:
A custom error handler. Log the error; show something presentable—ish—to the user.